marklogic-data-hub-central/ui/e2e/cypress/support/pages/browse.tsx

```diff
@@ -34,7 +34,7 @@ class BrowsePage {
   }
 
   clickPaginationItem(index: number) {
-    return cy.get(`#top-search-pagination-bar .ant-pagination-item-${index}`).click();
+    return cy.get(`#top-search-pagination-bar .ant-pagination-item-${index}`).click({force:true});
```

Comment:
It wasnt too hard to find a solution that will work without using `{force: true}`. We just have to look for it. Can you pls try "mouseleave" trigger if it works.

Reply:
mouseleave doesn't work but mouseout seems to work for me locally, I pushed it to verify if it passes on pipeline.

Reply:
Tests passed with the new changes, looks good to me.